esapi/views/profile.py

```diff
@@ -287,9 +287,9 @@ def transform_answer(self, es_query):
                     'timeTripInit': hit.expeditionStartTime.replace('T', ' ').replace('.000Z', ''),
                     'timeTripEnd': hit.expeditionEndTime.replace('T', ' ').replace('.000Z', ''),
                     'dayType': day_type_dict[hit.dayType],
-                    'valid': bool(hit.isValid)
+                    'valid': not bool(hit.notValid)
                 }
-                if not hit.isValid:
+                if not bool(hit.notValid):
```

Comment:
creo que el `not` no es necesario para realizar el conteo

Reply:
Cierto, lo corregiré